### ISSUE TITLE
Failed to deploy pods in kubernetes 1.4

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#6270]
**Rancher Version:**
master image
**Docker Version:**
1.12.2
**OS and where are the hosts located? (cloud, bare metal, etc):**
Ubuntu 16.04
**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
Single node rancher
**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**
Kubernetes 1.4
**Steps to Reproduce:**
Install kubernetes 1.4 from infra catalog
try to deploy any pod
**Results:** 
The pods are stuck in containercreating state and producing this error:

```
$ kubectl describe pods --namespace=testingress1
Name:		k8testrc1-8fcza
Namespace:	testingress1
Node:		hussein-10acre-1/104.197.240.246
Start Time:	Thu, 13 Oct 2016 19:22:37 +0200
Labels:		k8s-app=k8test1-service
Status:		Pending
IP:		
Controllers:	ReplicationController/k8testrc1
Containers:
  testcontainer:
    Container ID:	
    Image:		sangeetha/testnewhostrouting
    Image ID:		
    Port:		81/TCP
    State:		Waiting
      Reason:		ContainerCreating
    Ready:		False
    Restart Count:	0
    Volume Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-1pc7p (ro)
    Environment Variables:	<none>
Conditions:
  Type		Status
  Initialized 	True 
  Ready 	False 
  PodScheduled 	True 
Volumes:
  default-token-1pc7p:
    Type:	Secret (a volume populated by a Secret)
    SecretName:	default-token-1pc7p
QoS Class:	BestEffort
Tolerations:	<none>
Events:
  FirstSeen	LastSeen	Count	From				SubobjectPath	Type		Reason		Message
  ---------	--------	-----	----				-------------	--------	------		-------
  56m		56m		1	{default-scheduler }				Normal		Scheduled	Successfully assigned k8testrc1-8fcza to hussein-10acre-1
  56m		4s		142	{kubelet hussein-10acre-1}			Warning		FailedSync	Error syncing pod, skipping: failed to "SetupNetwork" for "k8testrc1-8fcza_testingress1" with SetupNetworkError: "Failed to setup network for pod \"k8testrc1-8fcza_testingress1(a2c3b262-9169-11e6-a468-02bcb92f6900)\" using network plugins \"cni\": failed to add IP addr to \"eth0\": file exists; Skipping pod"



Name:		k8testrc1-wn0yy
Namespace:	testingress1
Node:		hussein-10acre-3/104.198.54.251
Start Time:	Thu, 13 Oct 2016 19:22:37 +0200
Labels:		k8s-app=k8test1-service
Status:		Pending
IP:		
Controllers:	ReplicationController/k8testrc1
Containers:
  testcontainer:
    Container ID:	
    Image:		sangeetha/testnewhostrouting
    Image ID:		
    Port:		81/TCP
    State:		Waiting
      Reason:		ContainerCreating
    Ready:		False
    Restart Count:	0
    Volume Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-1pc7p (ro)
    Environment Variables:	<none>
Conditions:
  Type		Status
  Initialized 	True 
  Ready 	False 
  PodScheduled 	True 
Volumes:
  default-token-1pc7p:
    Type:	Secret (a volume populated by a Secret)
    SecretName:	default-token-1pc7p
QoS Class:	BestEffort
Tolerations:	<none>
Events:
  FirstSeen	LastSeen	Count	From				SubobjectPath	Type		Reason		Message
  ---------	--------	-----	----				-------------	--------	------		-------
  56m		56m		1	{default-scheduler }				Normal		Scheduled	Successfully assigned k8testrc1-wn0yy to hussein-10acre-3
  56m		58s		146	{kubelet hussein-10acre-3}			Warning		FailedSync	Error syncing pod, skipping: failed to "SetupNetwork" for "k8testrc1-wn0yy_testingress1" with SetupNetworkError: "Failed to setup network for pod \"k8testrc1-wn0yy_testingress1(a2c3e954-9169-11e6-a468-02bcb92f6900)\" using network plugins \"cni\": failed to add IP addr to \"eth0\": file exists; Skipping pod"
```

**Comment by galal-hussein** [comment]
Tested and worked on master image with kubernetes 1.4 deployed from the infra catalog.

**Comment by omegarus** [comment]
Does this fix included in v1.2.0-pre4-rc5 image ?
